apps/explorer/src/pages/object-result/views/ObjectView.tsx

```diff
@@ -257,18 +259,22 @@ export function ObjectView({ data }: ObjectViewProps) {
 				</div>
 			)}
 
-			<div style={{ gridArea: 'description' }}>
-				<DescriptionCard
-					name={name}
-					objectType={objectType}
-					objectId={data.data?.objectId!}
-					display={display}
-				/>
-			</div>
+			{objectId && (
```

Comment:
should this be centered horizontally within the div? 
<img width="890" alt="image" src="https://github.com/MystenLabs/sui/assets/122397493/035594b9-3b79-4750-8454-af70d6f2e960">

Reply:
@mamos-mysten Fix centering of the section